runtime/lib/ttnn/operations/eltwise/unary/unary.cpp

```diff
@@ -45,6 +46,21 @@ static void runEltwiseUnaryWithFastAndApproximateModeOp(
   tensorPool.insert_or_assign(op->out()->global_id(), out);
 }
 
+static void runEltwiseUnaryLeakyReluOp(
```

Comment:
It seems like in ttnn they have a specific macro for registering unary ops with float parameters:
`REGISTER_UNARY_OPERATION_WITH_FLOAT_PARAMETER` that covers multiple ops including leaky relu

I think we can rename this function to something like `runEltwiseUnaryWithFloatParameter` and also generalize the flatbuffers parameter table to something like:
```
table UnaryWithFloatParams {
  parameter: float;
}
```
This way we can reuse this code path for other unary ops that they register with `REGISTER_UNARY_OPERATION_WITH_FLOAT_PARAMETER`

Reply:
Yes, agree, great catch! I will update the code to reflect that.

Reply:
I updated the code to reflect this abstraction, please take a look @jnie-TT, I hope it should be good for now.